### PR TITLE
Kpobozhny labwork_1_5 completed

### PR DESCRIPTION
**Pull request by kpobozhny**

### CODE REVIEW

**Review by PBaranovskyi — APPROVED** [review]
All is good. Will merge next week in order to give other a chance to  do it by them own.

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_4.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        3
+        4
+        …
+        20*/
+
+public class labwork_1_5_4 {
+
+    public static void main(String[] args) {
+        int counter=0;
+        for (int i = 1; i < 300; i++) {
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_4.java]: Use ctrl+alt+L for formatting. PLease format your previous code

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    case 1: System.out.println("One");break;
+                    case 2: System.out.println("Two");break;
+                    case 3: System.out.println("Three");break;
+                    case 4: System.out.println("Four");break;
+                    case 5: System.out.println("Five");break;
+                    case 6: System.out.println("Six");break;
+                    case 7: System.out.println("Seven");break;
+                    case 8: System.out.println("Eight");break;
+                    case 9: System.out.println("Nine");break;
+                    default: if(x>9){System.out.println("Other"); }
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java]: what if x < 0?

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+        do{
+            System.out.print("Enter number x: ");
+            inputString = scan.nextLine();
+
+            try {
+                x = Integer.parseInt(inputString);
+                switch (x){
+                    case 1: System.out.println("One");break;
+                    case 2: System.out.println("Two");break;
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java]: format your code. CTRL+ALT+L

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                    case 3: System.out.println("Three");break;
+                    case 4: System.out.println("Four");break;
+                    case 5: System.out.println("Five");break;
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java]: Maybe it`s better to use one string variable assigned to a value in each case and print it with one System.out.println  after case in the end?.

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_5.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                // calculate SUM
+                for (int i = 0; i < n; i++) {
+                    sum+=i;
+                }
+
+                // calculate AVG
+                avg=(double)sum/n;
+
+                // print result
+                System.out.println("The Sum is:"+sum);
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_5.java]: format the code

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_7.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        boolean enteredCorrectValue=false;
+        Scanner scan = new Scanner(System.in);
+
+        do{
+            System.out.print("Enter number n: ");
+            inputString = scan.nextLine();
+
+            try {
+                n = Integer.parseInt(inputString);
+
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_7.java:28]: format the code

**Review by PBaranovskyi — CHANGES_REQUESTED** [review]

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java (hunk 2 above)
Comment (kpobozhny) [reply homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java]: I have mentioned this case in my comments .
//it was not described in the task what should be done if we have a negative number (x<1), so let's do nothing.

File: homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java (hunk 4 above)
Comment (kpobozhny) [reply homeworks/src/kpobozhny/homework/labwork_1_5/labwork_1_5_2.java]: yes, completely agree. fixed.

**Comment by kpobozhny** [comment]
Requested changes implemented.

**Pull request merged**